identity-service/src/notifications/index.js

```diff
@@ -190,15 +252,26 @@ class NotificationProcessor {
     }
 
     let startBlock = await getHighestBlockNumber()
-    logger.info(`Starting with ${startBlock}`)
+    logger.info(`Starting with block ${startBlock}`)
     await this.notifQueue.add({
       minBlock: startBlock,
       type: notificationJobType
     }, {
       jobId: `${notificationJobType}:${Date.now()}`
     })
 
-    await this.emailQueue.add({ type: 'unreadEmailJob' }, { jobId: Date.now() })
+    // TODO: Need to get current highest slot
+    // let startSlot = await getHighestSlot()
```

Comment:
Need to figure out a good way to get the latest slot.

Reply:
is this the highest slot processed for notifs? bc if so it should just be local to identity